functions/sample_function.ts

```diff
@@ -31,12 +36,35 @@ export const SampleFunctionDefinition = DefineFunction({
   },
 });
 
+// This function takes the input from the open form step, adds formatting, saves our
+// updated object into the Slack hosted Datastore, and returns the updated message.
 export default SlackFunction(
   SampleFunctionDefinition,
-  ({ inputs }) => {
-    const { message } = inputs;
-    const updatedMsg =
-      `:wave: You submitted the following message: \n\n>${message}`;
+  async ({ inputs, token }) => {
+    const sampleObject = {} as Record<string, unknown>;
+    sampleObject.original_msg = inputs.message;
+
+    // inputs.user is set from the interactivity_context we defined in sample_trigger.ts
+    // More info on interactivity here: https://api.slack.com/future/forms#add-interactivity
+    const updatedMsg = `:wave: ` + `<@${inputs.user}>` +
+      ` submitted the following message: \n\n>${inputs.message}`;
+    sampleObject.updated_msg = updatedMsg;
+
+    //initialize SlackAPI so we can use the Slack hosted Datastore
+    const client = SlackAPI(token, {});
+
+    //set unique object id, this will be our primary key in the Datastore
+    sampleObject.object_id = "OBJ-" +
+      await (Math.floor(100000 + Math.random() * 900000));
```

Comment:
I'd like to clean up this logic a bit, if possible. I believe that `uuid` is now part of the Deno standard library, so we should be able to make use of that. I believe we can also forgo the comment since the code should be self-describing at that point.